boards/silabs/dev_kits/xg22_ek2710a/xg22_ek2710a.dts

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright The Zephyr Project Contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <silabs/xg22/efr32mg22e224f512im40.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include <zephyr/dt-bindings/regulator/silabs_dcdc.h>
+#include "xg22_ek2710a-pinctrl.dtsi"
+
+/ {
+	model = "Silicon Labs BRD2710A (xG22 Explorer Kit)";
+	compatible = "silabs,xg22_ek2710a", "silabs,efr32mg22";
+
+	chosen {
+		zephyr,bt-hci = &bt_hci_silabs;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,console = &usart1;
+		zephyr,flash = &flash0;
+		zephyr,shell-uart = &usart1;
+		zephyr,sram = &sram0;
+		zephyr,uart-pipe = &usart1;
+	};
+
+	aliases {
+		led0 = &led0;
+		sw0 = &button0;
+		watchdog0 = &wdog0;
+
+		/* If enabled, MCUboot uses this for recovery mode entrance */
+		mcuboot-led0 = &led0;
+		mcuboot-button0 = &button0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led0: led_0 {
+			gpios = <&gpioa 4 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+
+		button0: button_0 {
+			gpios = <&gpioc 7 GPIO_ACTIVE_LOW>;
+			zephyr,code = <INPUT_KEY_0>;
+		};
+	};
+};
+
+&cpu0 {
+	clock-frequency = <76800000>;
+};
+
+&itm {
+	pinctrl-0 = <&itm_default>;
+	pinctrl-names = "default";
+	swo-ref-frequency = <DT_FREQ_K(76800)>;
+};
+
+&hfxo {
```

Comment:
This board also has a LFXO. Should it be enabled and selected for all the LF clock trees?

Reply:
Good catch, added the lfxo and selected the clock trees